### ISSUE TITLE
FBInstant.payments.purchaseAsync Error

### ISSUE THREAD

**Issue by yucccc** [fbsamples/fbinstant-samples#31]
I use FBInstant.payments.purchaseAsync and this error occurs
![image](https://user-images.githubusercontent.com/24189407/127427965-7e9bd724-15df-42f3-9faa-ba82dccce68b.png)
The version used is https://connect.facebook.net/en_US/fbinstant.6.2.js

The demo used is the code provided in GitHub

**Comment by yucccc** [comment]
My local development method is as follows:
1. Open https link For example: https://localhost:8000
2. Then embed the link in https://www.facebook.com/embed/instantgames/YOUR_APPID/player?game_url=<My link https://localhost:8000>
3. The above problem occurs when you click to pay

**Comment by AlanWilms** [comment]
Hi @yucccc,

Just to get a better understanding, did you already call `FBInstant.initializeAsync()`, `FBInstant.startGameAsync()`, and `FBInstant.payments.onReady()` before `FBInstant.payments.purchaseAsync`?

**Comment by yucccc** [comment]
@AlanWilms 

Yes, I follow the normal call sequence, I am using the official demo. This error looks like some method is incompatible. It shows a pop-up window that is loading for a while, and then it disappears and an error is reported.

`setActiveWrapper is undefined in ModalLayer`

I tried Google Chrome and Firefox are the same

**Comment by yucccc** [comment]
@AlanWilms 
I found a related question in the community 
https://developers.facebook.com/community/threads/492959575110030/ 
but it was not answered

**Comment by AlanWilms** [comment]
@yucccc 

Thanks for providing the details. Let me investigate further and get back to you afterwards

**Comment by AlanWilms** [comment]
![in-app-purchases test](https://user-images.githubusercontent.com/19272026/128800630-9c6c3ffa-46bd-41ed-b41b-97e9027bef72.png)
I tried with the instructions for both the hosted version (`yarn upload --project in-app-purchases`) and the local version (`yarn run-dist --project in-app-purchases`) after setting up a payouts account in developers.facebook.com and adding the app_id to the config.json and was unable to reproduce the issue.

**Comment by RuinErickai** [comment]
When I was testing the payment, it popped up like this  
![image](https://github.com/fbsamples/fbinstant-samples/assets/52850984/25347a77-fcba-42fe-b10c-cbd2c86e18a4)
How do I fix this、
